dts/arm/st/u0/stm32u031X4.dtsi

```diff
@@ -7,18 +7,6 @@
 #include <st/u0/stm32u031.dtsi>
 
 / {
-	sram1: memory@20000000 {
-		compatible = "zephyr,memory-region", "mmio-sram";
-		reg = <0x20000000 DT_SIZE_K(8)>;
-		zephyr,memory-region = "SRAM1";
-	};
-
-	sram2: memory@20080000 {
-		compatible = "zephyr,memory-region", "mmio-sram";
-		reg = <0x20080000 DT_SIZE_K(4)>;
-		zephyr,memory-region = "SRAM2";
-	};
-
```

Comment:
@erwango, I wonder if factorizing SRAMs size info in **stm32u0??.dtsi** sub-series files is a good idea. Maybe it is more convenient to have both flash and SRAM sizes all defined together in **stm32u??X?.dtsi**, for consistency with the other STM32 SoCs.
Based on feedback on this P-R related to STM32U0xx, I plan to propose equivalent changes in other SoCs to better name SRAM nodes accrdong to reference manual wording.